.github/workflows/docs.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Install dependencies
       run: |
         python3 -m pip install --upgrade pip
-        python3 -m pip install -e '.[docs]'
+        python3 -m pip install -e '.[docs, pulse]'
```

Comment:
The `pulse` extra shouldn't be required to build docs now that it's been added autodoc mock imports